iac/terraform/aws/eventbridge/eventbridge.tf

```diff
@@ -13,15 +13,11 @@ resource "aws_pipes_pipe" "donation_request_pipe" {
     filter_criteria {
       filter {
         pattern = jsonencode({
-          eventName : ["INSERT", "MODIFY"],
-          dynamodb : {
-            NewImage : {
-              PK : {
-                S : [{ prefix : "BLOOD_REQ#" }]
-              },
-              SK : {
-                S : [{ prefix : "BLOOD_REQ#" }]
-              }
+          "eventName" : ["INSERT", "MODIFY"],
+          "dynamodb" : {
+            "NewImage" : {
+              "PK" : { "S" : [{ "prefix" : "BLOOD_REQ#" }] },
+              "SK" : { "S" : [{ "prefix" : "BLOOD_REQ#" }] }
```

Comment:
Is it possible to take these values `BLOOD_REQ` in the variable and then use it?

Reply:
we can. we will have to create a local variable. but it looks unnecessary.